tests/shanghai/eip4895_withdrawals/test_withdrawals.py

```diff
@@ -31,763 +22,352 @@
-        ),
-        # Withdrawal with value
-        Withdrawal(
-            index=1,
-            validator_index=0,
-            address=empty_accounts[1],
-            amount=1,
-        ),
-        # Withdrawal with maximum amount
-        Withdrawal(
-            index=2,
-            validator_index=0,
-            address=empty_accounts[2],
-            amount=2**64 - 1,
+    withdrawal = Withdrawal(
+        index=0,
+        validator_index=0,
+        address=Address(0x01),
+        amount=1,
+    )
+
+    blocks = [
+        Block(
+            withdrawals=[withdrawal],
         ),
     ]
-    all_post = {
-        empty_accounts[0]: Account.NONEXISTENT,
-        zero_balance_contract: Account(code=Op.STOP, balance=0),
-        empty_accounts[1]: Account(balance=ONE_GWEI),
-        empty_accounts[2]: Account(balance=(2**64 - 1) * ONE_GWEI),
-    }
 
-    withdrawals: List[Withdrawal] = []
-    post: Mapping[Address, Account | object] = {}
-    if test_case == ZeroAmountTestCases.TWO_ZERO:
-        withdrawals = all_withdrawals[0:2]
-        post = {
-            account: all_post[account]
-            for account in post
-            if account in [empty_accounts[0], zero_balance_contract]
-        }
-    elif test_case == ZeroAmountTestCases.THREE_ONE_WITH_VALUE:
-        withdrawals = all_withdrawals[0:3]
-        post = {
-            account: all_post[account]
-            for account in post
-            if account
-            in [
-                empty_accounts[0],
-                zero_balance_contract,
-                empty_accounts[1],
-            ]
-        }
-    elif test_case == ZeroAmountTestCases.FOUR_ONE_WITH_MAX:
-        withdrawals = all_withdrawals
-        post = all_post
-    elif test_case == ZeroAmountTestCases.FOUR_ONE_WITH_MAX_REVERSED:
-        for i, w in enumerate(reversed(all_withdrawals)):
-            withdrawals.append(
-                Withdrawal(
-                    index=i,
-                    validator_index=w.validator_index,
-                    address=w.address,
-                    amount=w.amount,
-                )
-            )
-        post = all_post
-
-    blockchain_test(
-        pre=pre,
-        # TODO: Fix in BlockchainTest? post: Mapping[str, Account | object]
-        # to allow for Account.NONEXISTENT
-        post=post,
-        blocks=[Block(withdrawals=withdrawals)],
-        tag=test_case.value,
-    )
+    # Call to non-existent contract succeeds but does nothing
+    post: Dict[str, Any] = {}
+
+    blockchain_test(pre=pre, post=post, blocks=blocks)
 
 
-@pytest.mark.xdist_group(name="bigmem")
-def test_large_amount(
+def test_withdrawal_zero_amount(
     blockchain_test: BlockchainTestFiller,
     pre: Alloc,
 ) -> None:
     """
-    Test withdrawals that have a large gwei amount.
-
-    Test such that that (gwei * 1e9) could overflow uint64 but not uint256.
+    Test withdrawal with zero amount.
+    Edge case: system call with zero-value withdrawal.
```

Comment:
You should record the values in the contract otherwise you are not testing something new. Even the usefulness of this case for us is debatable, as we just constuct a call with whatever the withdrawals is and call the contract